<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1310</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="takePhotoButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>550</y>
     <width>170</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Take photo</string>
   </property>
  </widget>
  <widget class="QLabel" name="camLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="photoLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>50</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1310</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Centaur</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>facEmotion</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="savePhotoButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>550</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save photo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
